__tests__/app.test.js

```diff
@@ -52,20 +52,62 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  test("GET 200: responds with an array of all articles sorted by date in descending order, with body property removed", () => {
```

Comment:
So your test description technically contains three behaviours
1. all articles returned
2. body property isn't there
3. its sorted by date in desc order

Please amend the test desc to reflect what you're actually checking in this block. I would also consider making the descending order check a separate test as it is an additional behaviour.
Also are you actually checking that there is **no .body property on each article**?